src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Config/FluentValidationActionFilterTests.cs

```diff
@@ -0,0 +1,389 @@
+#nullable enable
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using System.Threading.Tasks;
+using FluentValidation;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.WebUtilities;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Config;
+
+public class FluentValidationActionFilterTests : IntegrationTest<TestStartup>
+{
+    private const string NotEmptyCode = "NotEmpty";
+    private const string NotEmptyMessage = "Must not be empty.";
+
+    private FluentValidationActionFilterTests(TestApplicationFactory<TestStartup> testApp) : base(testApp)
+    {
+    }
+
+    public class TestClassTests : FluentValidationActionFilterTests
+    {
+        public TestClassTests(TestApplicationFactory<TestStartup> testApp) : base(testApp)
+        {
+        }
+
+        [Fact]
+        public async Task Body_Valid()
+        {
+            var body = new TestClass { Name = "Test name" };
+
+            var client = BuildApp().CreateClient();
+            var response = await client.PostAsJsonAsync(nameof(TestController.TestClassBody), body);
+
+            response.AssertOk();
+        }
+
+        [Fact]
+        public async Task Body_Invalid()
```

Comment:
This looks like a valid scenario to me? Maybe just need to change the name of the test